exist-core/src/test/xquery/xquery3/json-to-xml.xql

```diff
@@ -0,0 +1,49 @@
+xquery version "3.1";
+
+(:~
+ : Test various F+O functions
+ :)
+module namespace jsonxml="http://exist-db.org/xquery/test/jsonxml";
+
+import module namespace test="http://exist-db.org/xquery/xqsuite" at "resource:org/exist/xquery/lib/xqsuite/xqsuite.xql";
+
+
+declare
+    %test:assertEquals("<fn:map xmlns:fn='http://www.w3.org/2005/xpath-functions'><fn:number key='x'>1</fn:number><fn:array key='y'><fn:number>3</fn:number><fn:number>4</fn:number><fn:number>5</fn:number></fn:array></fn:map>")
```

Comment:
during development of the mongodb and couchbase drivers, I found several issues while parsing complexer structures (deeper, nested). You might want to test these as well.

Reply:
If we can get the XQuery test suite runner working under eXist 5.0.0, I think we could be confident: https://github.com/eXist-db/exist-xqts-runner/issues/7.